apps/game_backend/lib/game_backend/users.ex

```diff
@@ -233,11 +260,43 @@ defmodule GameBackend.Users do
     end
   end
 
+  @doc """
+  Level up the Dungeon Settlement of a user.
+
+  Returns the updated user if the operation was successful.
+  """
+  def level_up_dungeon_settlement(user_id, level_up_costs) do
```

Comment:
Isn't this the same as `level_up_kaline_tree`? just changing the `increment_settlement_level`right?